spec/event_builder_spec.rb

```diff
@@ -182,7 +182,6 @@
   it 'should create a valid Event when create_conversion_event is called with revenue event tag' do
     event_tags = {'revenue' => 4200}
 
-    @expected_conversion_params[:visitors][0][:attributes] = []
```

Comment:
what was the reason it was initialized before but stale in some of the methods I am seeing. 
@oakbani